sympy/physics/vector/tests/test_point.py

```diff
@@ -126,3 +127,97 @@ def test_point_partial_velocity():
     assert p.partial_velocity(N, u1) == A.x
     assert p.partial_velocity(N, u1, u2) == (A.x, N.y)
     raises(ValueError, lambda: p.partial_velocity(A, u1))
+
+def test_point_vel(): #Basic functionality
+    q1, q2 = dynamicsymbols('q1 q2')
+    N = ReferenceFrame('N')
+    B = ReferenceFrame('B')
+    Q = Point('Q')
+    O = Point('O')
+    Q.set_pos(O, q1 * N.x)
+    raises(ValueError , lambda: Q.vel(N)) #Velocity of Q is not defined
+    O.set_vel(N, q2 * N.y)
+    assert O.vel(N) == q2 * N.y
```

Comment:
Now sure why you are checking O. I think you meant:

```
Q.vel(N) == q2*N.y + q1.diff()*N.x
```

Reply:
This funct is checking basics like if vel is set using set_vel() then vel() returns it or not.